src/sentry/models/notificationsetting.py

```diff
@@ -122,5 +132,51 @@ class Meta:
     )
 
 
+@control_silo_only_model
+class UserNotificationSetting(Model, NotificationSettingBase):
```

Comment:
With this model in place, would the process to use the separated models be something like:

1. start dual writing to both `UserNotificationSetting` and `NotificationSetting` based on the actor type.
2. backfill user settings.
3. update reads

Or did you have another transition plan in mind?

Reply:
Yes, exactly.
Which table to write depends on the scope type instead of the actor type. All records with scope `USER` will be in `UserNotificationSetting` and the rest (`TEAM`, `PROJECT` and `ORGANIZATION`) will be in `NotificationSetting`.